test/imageext_test.py

```diff
@@ -37,7 +36,6 @@ def test_save_bad_filename(self):
         u = u"a\x00b\x00c.png"
         self.assertRaises(pygame.error, save_extended, im, u)
 
-    @expected_error(TypeError)
     def test_load_bad_filename(self):
         u = u"a\x00b\x00c.png"
         self.assertRaises(pygame.error, load_extended, u)
```

Comment:
I'm a bit confused why these no longer raise an error given that they seem to do crazy things (load an empty list and an invalid filename)?